decidim-core/app/types/decidim/core/participatory_space_interface.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Core
+    # This type represents a ParticipatoryProcess.
+    ParticipatorySpaceInterface = GraphQL::InterfaceType.define do
+      name "ParticipatorySpaceInterface"
+      description "The interface that all participatory spaces should implement."
+
+      field :id, !types.ID, "The participatory space's unique ID"
+
+      field :title, !TranslatedFieldType, "The name of this participatory space."
+
+      field :allComponents, types[ComponentInterface] do
```

Comment:
`field :components`, maybe?

Reply:
You're probably right. The `allWhatever` convention seems to be generally only used in top-level fields.